init.lua

```diff
@@ -115,4 +115,4 @@ function mon_meta:value(player)
 end
 
 dofile(modpath .. "standard_monoids.lua")
-dofile(modpath .. "test.lua")
+dofile(modpath .. "test.lua")
```

Comment:
I don't think you should remove newlines from the last line of every file. Newlines at the end of files is part of the POSIX standard for text files, so removing them would make the files technically not portable (though all editors will support it anyway). More importantly, the newlines will get readded when someone changes the files from a POSIX-compliant system.